qt/applications/workbench/workbench/projectrecovery/test/test_projectrecovery.py

```diff
@@ -87,10 +87,8 @@ def test_remove_empty_dir(self):
         self.assertTrue(not os.path.exists(empty))
 
     def test_remove_empty_dir_throws_outside_of_workbench_directory(self):
-        path = os.path.join(self.working_directory, "test.txt")
-        open(path, 'a').close()
-
-        self.assertRaises(RuntimeError, self.pr._remove_empty_folders_from_dir(path))
+        os.mkdir(os.path.join(self.working_directory, "temp"))
```

Comment:
Will this temporary directory be removed?

Reply:
It's inside a cleaned up directory, so yes.

Reply:
:+1: